shop/src/main/java/ro/msg/learning/shop/model/Customer.java

```diff
@@ -0,0 +1,32 @@
+package ro.msg.learning.shop.model;
+
+import lombok.*;
+import javax.persistence.*;
+import java.util.List;
+
+@Builder
+@Entity
+@Table(name = "Customer")
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode
+public class Customer {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name="ID")
+    private Integer id;
```

Comment:
You should create a BaseEntity class where you can declare your id and every model can then extend that class. Basically, each model has the id field, right?